jclouds-plugin/src/main/java/jenkins/plugins/jclouds/compute/JCloudsCloud.java

```diff
@@ -475,4 +514,79 @@ public FormValidation doCheckEndPointUrl(@QueryParameter String value) {
             return FormValidation.ok();
         }
     }
+
+    public static class ConverterImpl extends XStream2.PassthruConverter<JCloudsCloud> {
```

Comment:
This is not a public api, if it needs to be public to take effect it should be `@Restricted(DoNotUse.class)`.